AndroidAnnotations/androidannotations-rest-spring/rest-spring/src/main/java/org/androidannotations/rest/spring/handler/HeadHandler.java

```diff
@@ -39,7 +39,9 @@ public void validate(Element element, ElementValidation validation) {
 
 		restSpringValidatorHelper.hasHttpHeadersReturnType((ExecutableElement) element, validation);
 
-		restSpringValidatorHelper.urlVariableNamesExistInParametersAndHasNoOneMoreParameter((ExecutableElement) element, validation);
```

Comment:
These three validations are present in multiple handlers, what about extracting to a method?